maze_transformer/generation/latticemaze.py

```diff
@@ -14,8 +14,10 @@
     adjlist_end="<ADJLIST_END>",
     target_start="<TARGET_START>",
     target_end="<TARGET_END>",
-    start_path="<START_PATH>",
-    end_path="<END_PATH>",
+    origin_start="<ORIGIN_START>",
+    origin_end="<ORIGIN_END>",
+    path_start="<START_PATH>",
+    path_end="<END_PATH>",
```

Comment:
agh! thank you! this had been bugging me as well.

Do you think it's worth changing the string as well so that start/end is the suffix?

Reply:
Yeah if people are happy to have their trained models not be compatible with future datasets

Reply:
Worth asking on slack?

Reply:
Actually I just realised that adding the origin isn't backwards compatible either so I'll just go for it and announce it